app/src/androidTest/java/org/kiwix/kiwixmobile/help/HelpActivityTest.kt

```diff
@@ -38,7 +38,6 @@ class HelpActivityTest : BaseActivityTest<HelpActivity>() {
       clickOnWhereIsContent()
       assertWhereIsContentIsExpanded()
       clickOnWhereIsContent()
-      clickOnLargeZimFiles()
       assertLargeZimsIsExpanded()
       clickOnLargeZimFiles()
```

Comment:
and the 2 lines below that deletion. The test expands then collapses the sections